File: proyectos/java/CursoPOOUber/Java/Driver.java
public class Driver extends Account{    
    
    public Driver(String name, String document, String email, String password){
        super(name, document, email, password);

    }
     void printDataDriver(){
        System.out.println("Document Driver: "+document+"Name: "+
        name+" Email: "+email+" password: "+password);
    }
    
}

File: proyectos/java/CursoPOOUber/PHP/driver.php
<?php
require_once('account.php');

class Driver extends Account{

    public function __construct($id,$name,$document,$email,$password){
        parent:: __construct($name,$document);
        $this -> id = $id;
        $this -> email = $email;
        $this -> password = $password;
    }
    public function PrintDataDriver(){
        echo "ID:$this->id, DRIVER:{$this->name->name}, EMAIL:{$this->email->email},DOCUMENT{$this -> password}";  
    }
}